box-share-sdk/src/main/res/values/styles.xml

```diff
@@ -38,9 +38,10 @@
         <item name="android:textColor">@color/box_sharesdk_background</item>
         <item name="android:textAppearance">@style/TextAppearance.AppCompat.Button</item>
     </style>
+
+    <!--Since we changed gray color to white, gray buttons don't look good on phones older than v21
+    Using default blue buttons for the same-->
     <style name="RaisedGrayButton"  parent="@style/RaisedButton">
```

Comment:
Remove gray button and just use Raisedbutton (unless this is styled in styles-21)